### DISCUSSION TITLE
Particles Glitch on SVG Path with Parallax enabled

### DISCUSSION THREAD

**Discussion by motion-work** [matteobruni/tsparticles#1575] [Q&A]
I have the following SVG Path where the particles move within: [Demo](https://res.cloudinary.com/delicious-cookie/video/upload/v1622468152/kaleido/Screen_Recording_2021-05-31_at_15.17.19_qstlzo.mov)

Also I enabled the parallax effect:

```js
onHover: {
    enable: true,
    mode: "bubble",
    parallax: {
        enable: true,
        force: 250,
        smooth: 20,
    },
},
```

When I move my mouse to the right half of the screen the particles reach the border of the SVG. therefore they start to glitch. Is there a way to fix that? I have also tried a different SVG Shape but with no success.

**Comment by matteobruni** [comment]
I need to debug it, it looks like an issue that needs a fix to the polygon mask plugin. I'll let you know when I have some news.

**Reply by motion-work** [reply]
you're the best. Here is the exact SVG Shape I am using: [SVG Path](https://res.cloudinary.com/delicious-cookie/image/upload/v1620726994/kaleido/form_real__djcmpj.svg)

**Reply by motion-work** [reply]
@matteobruni any updates on this?

**Reply by matteobruni** [reply]
@motion-work not yet sorry, I made some debug but I didn't find the problem yet. I thought this was easier.
